### PR TITLE
Symfony Flex compatibility, Scalar Types etc.

### PR DESCRIPTION
**Pull request by vinzenz-muser**
The command works now for Symfony Flex as well.

### CODE REVIEW

File: Command/SyncProjectCommand.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         $io = new SymfonyStyle($input, $output);
 
         // Read configuration
-        $config = $this->getConfigurationForEnvironment($input->getArgument('environment'));
+        $config = $this->getConfigurationForEnvironment(strval($input->getArgument('environment')));
```
Comment (sheeep) [inline Command/SyncProjectCommand.php]: `(string)`

File: Command/SyncProjectCommand.php [hunk 2]
```diff
@@ -236,29 +231,23 @@ private function hasEnvFileSupport($env = 'local')
...
-        if (!$hasEnvFileSupport) {
-            $file = 'local' === $environment ?
-                sprintf('%s/config/parameters.yml', $this->rootDir) :
-                sprintf('%s/config/parameters.%s.yml.dist', $this->rootDir, $environment)
-            ;
-
-            if (!file_exists($file)) {
-                throw new LogicException(sprintf('No parameters file for environment "%s" found.', $environment));
-            }
+        $file = sprintf('%s/../../.env.local', $this->rootDir);
```
Comment (sheeep) [inline Command/SyncProjectCommand.php]: Is this really true? It seems to me, that `rootDir` is either `projectDir` or `projectDir/src`. Either way `../..` would result in a wrong directory, no?

**Review by sheeep — CHANGES_REQUESTED** [review]
Can you check on the comments, the rest LGTM.
Thank you either way!

File: Command/SyncProjectCommand.php (hunk 1 above)
Comment (vinzenz-muser) [reply Command/SyncProjectCommand.php]: Fixed

File: Command/SyncProjectCommand.php (hunk 2 above)
Comment (vinzenz-muser) [reply Command/SyncProjectCommand.php]: rootDir was `~/dev/naturmetropole-graubuenden/src/HttpKernel/` so this version worked, however `projectDir` is the better choice anyway.

**Pull request merged**